.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Somewhat yes, as this provides the VSC setting to ensure the code is PEP8 compliant. It can be removed if preferred by the team, however. And yes, we can add black if you would like.

Reply:
Going to remove the .vscode folder from the branch until the team can agree on what should go in it.